src/common/validations/isLayerNameExists.ts

```diff
@@ -0,0 +1,10 @@
+import { IMapProxyJsonDocument } from '../interfaces';
+
+export function isLayerNameExists(jsonDocument: IMapProxyJsonDocument, layerName: string): boolean {
```

Comment:
if the method name is "isLayerNameExists" it should return false it the layer doesn't exit not throw an exception

For that you should choose other method name such as validate....

Reply:
it is returns flase it layer doesn't exists : 
 return publishedLayers.includes(layerName); => includes() returns true/false, the the error throw is from the try catch section